imagersite/imager_profile/handlers.py

```diff
@@ -0,0 +1,27 @@
+from django.dispatch import receiver
+from django.db.models.signals import post_save, post_delete
+from django.contrib.auth.models import User
+from .models import ImagerProfile
+
+
+@receiver(post_save, sender=User)
+def create_user_profile(sender, **kwargs):
+    instance = kwargs.get('instance')
+    if not instance or kwargs.get('raw', False):
+        return
+    try:
+        instance.profile
+    except:
```

Comment:
There is a specific error you're looking for here.  It's `ImagerProfile.DoesNotExist`.